src/web/app/pages-student/student-course-details-page/student-course-details-page.component.scss

```diff
@@ -7,7 +7,12 @@
 }
 
 .profile-pic {
-  height: 35%;
+  width: 85%;
+  height: 50%;
```

Comment:
Would it be better to use `max-width: ?` and `max-height: ?` to maintain aspect ratio of image?